grpc-server.go

```diff
@@ -720,24 +720,15 @@ func (multi *MultiEpoch) StreamTransactions(params *old_faithful_grpc.StreamTran
 	}
 	gsfaReader, _ := multi.getGsfaReadersInEpochDescendingOrderForSlotRange(ctx, startSlot, endSlot)
```

Comment:
may need to add a guard if the gsfa index was loaded or not?

Reply:
see method-getSignaturesForAddress

```if len(gsfaIndexes) == 0 {
		return &jsonrpc2.Error{
			Code:    jsonrpc2.CodeInternalError,
			Message: "getSignaturesForAddress method is not enabled",
		}, fmt.Errorf("no gsfa indexes found")
	}```

Reply:
Possible add back an optional slow path AccountInclude in filter transaction in case gSFA is not available.